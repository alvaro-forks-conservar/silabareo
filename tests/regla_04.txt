aprobar
ácrata
híbrido
ogro
aplacar
reclamar
bugle
efluvio
afrenta
ablandar