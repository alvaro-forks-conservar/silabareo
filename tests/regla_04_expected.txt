a-pro-bar
á-cra-ta
hí-bri-do
o-gro
a-pla-car
re-cla-mar
bu-gle
e-flu-vio
a-fren-ta
a-blan-dar